packages/rolldown/tests/fixtures/builtin-plugin/glob-import/vite/_config.ts

```diff
@@ -0,0 +1,31 @@
+import { importGlobPlugin } from 'rolldown/experimental'
+import { RolldownOutput } from 'rolldown'
+import { defineTest } from 'rolldown-tests'
+import { expect } from 'vitest'
+import * as fs from 'node:fs'
+import * as path from 'node:path'
+
+// ported from https://github.com/vitejs/vite/tree/021443c5a11eedfb176f2c65d5fab65e9c21adfb/packages/vite/src/node/__tests__/plugins/importGlob
+export default defineTest({
+  config: {
+    input: './fixture-a/index.ts',
+    plugins: [
```

Comment:
Could you add `output.chunkFilenames: '[name].js'`, it could avoid generate hash at snapshot and it avoid update it if hash changed.

Reply:
Nice, changed it.